pages/resources.css

```diff
@@ -0,0 +1,5 @@
+/* TODO: determine how to put a gutter in between the resources */
+.resourcesWrapper {
```

Comment:
Didn't want to go down a CSS side path while core page functionality is still in progress.

Reply:
Feel free to start adding CSS! We'll need to add margin between every resource item, for one.

Reply:
I moved the className to the outermost element in the ResourceCard, otherwise the margins I was passing in weren't having any effect. Figured it was safe to do since the component wasn't in use anywhere else yet.